packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulators-sell-button.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { getCardLabels } from '@deriv/shared';
+import { Button, Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+
+type TAccumSellButton = {
+    current_stake: number | null;
+    currency?: string;
+    is_disabled: boolean;
+    onClick: (e: React.MouseEvent<HTMLElement>) => void;
+};
+const AccumulatorsSellButton = ({ current_stake, currency, is_disabled, onClick }: TAccumSellButton) => {
```

Comment:
Not sure about naming, because it's also containing `<Fieldset />` with specific className.